merge_sort.py

```diff
@@ -0,0 +1,39 @@
+def mersort(unslist):
+    """Return a sorted list from a list.
+
+    Implementation after psuedocode here:
+    https://en.wikipedia.org/wiki/Merge_sort
+    """
+    if len(unslist) <= 1:
+        return unslist
+    lunslist = unslist[:len(unslist)/2]
+    runslist = unslist[len(unslist)/2:]
```

Comment:
1: rather than doing this math twice, set a local var.
2: use floored division.  In Py3 this will fail for lists of odd length because your index will be a float.